README.md

```diff
@@ -150,3 +150,33 @@ The following parameters affect high availability:
 
   - `vip.vip` - Which IP to use as a VIP that is traded between the
     two nodes.
+
+### HA Failure Modes
+
+Our HA solution is focused on preventing downtime in the face of
+upgrades or other single-node failure. As such, we do not attempt to
+solve scenarios where the two databases cannot communicate with one
+another (e.g. network failure). In this case, it is possible that the
+replica believes the master to be down, and will promote itself to be
+master. The Postgres servers are then in a state of "split-brain" and
+requests to the DB will be split between the two nodes.
+
+To mitigate this, each node checks to see who is master. If both
+nodes are master (split-brain), both immediately shut down to prevent
+inconsistent data states. *This will result in downtime*. But we
+downtime is preferable over inconsistent database states.
```

Comment:
The wording here (on the break) is weird.  "But we downtime is preferable..."

I think you accidentally a word.